mgnify_pipelines_toolkit/analysis/assembly/summarise_antismash_bgcs.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2024-2025 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import fileinput
+import logging
+import pandas as pd
+import os
+import re
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s: %(message)s"
+)
+
+"""
+Script parses antismash GFF output and adds descriptions from glossary https://docs.antismash.secondarymetabolites.org/glossary/.
+
+Descriptions took from antismash GitHub repo under documentation/docs/glossary.md
+"""
```

Comment:
This should be in the script help IMO, it's useful to have this prinnted when calling "--help"